test/suites/zombie_data_preservers/test_zombie_data_preservers.ts

```diff
@@ -1,19 +1,21 @@
 import "@tanssi/api-augment";
 
 import { beforeAll, describeSuite, expect } from "@moonwall/cli";
-import {
-    ALITH_ADDRESS,
-    BALTATHAR_ADDRESS,
-    BALTATHAR_PRIVATE_KEY,
-    CHARLETH_ADDRESS,
-    type KeyringPair,
-} from "@moonwall/util";
+import { BALTATHAR_ADDRESS, BALTATHAR_PRIVATE_KEY, CHARLETH_ADDRESS, type KeyringPair } from "@moonwall/util";
 import { ApiPromise, Keyring, WsProvider } from "@polkadot/api";
 import { u8aToHex } from "@polkadot/util";
 import { decodeAddress } from "@polkadot/util-crypto";
 import { WebSocketProvider, ethers, parseUnits } from "ethers";
-import fs from "node:fs/promises";
-import { getHeaderFromRelay, getTmpZombiePath, signAndSendAndInclude, sleep } from "utils";
+import { getHeaderFromRelay, getTmpZombiePath, signAndSendAndInclude, waitForLogs } from "utils";
+
+// Checks every second the log file to find the watcher best block notification until it is found or
+// timeout is reached. If timeout is reached, throws an error.
+export async function waitForLogs2(logFilePath: string, timeout: number, logs: string[]): Promise<void> {
```

Comment:
we need either include waitLogs to this function and call it waitLogs or come up with a better name, because I don't think it's good to have function like waitLogs, waitLogs2, etc

Reply:
Yeah this was a temporary name, will rename it to assertWaitForLogs or similar